tests/conftest.py

```diff
@@ -2,34 +2,35 @@
 #
 # SPDX-License-Identifier: MIT
 
-import os
 import shutil
 
 from pathlib import Path
+from tempfile import TemporaryDirectory
 
 import pytest
 
 
-@pytest.fixture(name="project_directory_uri")
-def _project_directory_uri(tmp_path):
-    leading_slashes = "//" if os.sep == "/" else "///"
-    return f"file:{leading_slashes}{tmp_path.as_posix()}/plugin"
+@pytest.fixture(scope="session")
+def plugin_dir():
+    with TemporaryDirectory() as d:
```

Comment:
I was gonna ask why you don't use tmp_path here, but it's about fixture re-use and session scope – correct?

Sorry for not testing myself; I'm on phone, literally gonna to bed rn. :)

Reply:
Yup used for an entire session!